crates/autopilot/src/arguments.rs

```diff
@@ -113,15 +113,6 @@ pub struct Arguments {
     )]
     pub max_surplus_fee_age: Duration,
 
-    /// The interval between successive limit order surplus fee updating loops in seconds.
-    #[clap(
-        long,
-        env,
-        default_value = "60",
-        value_parser = shared::arguments::duration_from_seconds,
-    )]
-    pub surplus_fee_update_interval: Duration,
```

Comment:
**Edit**: I see your PR comment about not ever needing to change it. I think since it was already there it wouldn't hurt to have kept it, but I also see your point that it isn't clear if we will really change it ever.

Reply:
I prefer not having unnecessary configuration. It makes the arguments overall harder to understand and there is more code moving the config around. Regardless of whether the argument was already there, we should independently judge whether it was useful.

With the new loop I don't see no reason to configure this argument because it no longer affects throughout. It did in the previous version which made it important there.

Reply:
Makes sense to me.